src/utils/AdrValidation.ts

```diff
@@ -19,7 +19,7 @@ export const adrValidation = Joi.object().keys({
   }).required(),
   permittedDangerousGoods: Joi.array().items(Joi.string()).min(1).required(),
   compatibilityGroupJ: Joi.boolean().optional().allow(null),
-  additionalExaminerNotes: Joi.string().optional().allow(null),
+  additionalExaminerNotes: Joi.string().max(1024).optional().allow(null),
```

Comment:
this is not in the scope of the ticket. Can you confirm why it is done here?

Reply:
This was missed out when we implemented ADR and I spoke with Umar and he said to do it in this ticket as it's a minor thing